cEP-0020.md

```diff
@@ -0,0 +1,167 @@
+# Newcomer metrics and Gamification
+
+| Metadata |                                                        |
+| -------- | ------------------------------------------------------ |
+| cEP      | 20                                                     |
+| Version  | 1.0                                                    |
+| Title    | Newcomer metrics and Gamification system               |
+| Authors  | Shrikrishna Singh <mailto:krishnasingh.ss30@gmail.com> |
+| Status   | Proposed                                               |
+| Type     | Process                                                |
+
+## Abstract
+
+This cEP describe how a newcomer at coala will become a developer on the
+basis of their activities in the community.
+
+For the implementation of this process, we will build a gamification system.
+So, every newcomer at coala have to complete the processes involved in the gamification system in order to become a developer.
+
+## Background
+
+### Newcomer Experience
+
+- How to attract more newcomers
+  - How to cope with the increasing amounts of newcomers
+- How to teach and help newcomers scalable so we can keep working on coala.
```

Comment:
>help newcomers **scalable**

sentence doesn't make sense to me

Reply:
Maybe help newcomers to learn and grow faster which can help the community in getting more new contributors?

Actually, I copied the background section from [here](https://github.com/coala/teams/blob/master/community/community_ideas.md#newcomer-experience)